src/components/Book.js

```diff
@@ -9,31 +9,38 @@ export default function Book(props) {
     <li className="bookelement" id={propsHolder.book[0]}>
       <div className="leftdiv">
         <div>
-          <p>{propsHolder.book[1][0].category}</p>
-          <p>{propsHolder.book[1][0].title}</p>
-          <p>{propsHolder.book[1][0].author}</p>
+          <p className="category">{propsHolder.book[1][0].category}</p>
+          <p className="title">{propsHolder.book[1][0].title}</p>
+          <p className="author">{propsHolder.book[1][0].author}</p>
```

Comment:
- [ ] Great job working on the styles so far but I have some notes I noticed need to be fixed to match the Zeplin styles as much as possible here as I can see this part is not the same you need to fix some spacing here so kindly add some margin/padding to the divs here and align the elements well to be the same as Zeplin design 🔥🚀🚀♻♻

- Your design
![image](https://user-images.githubusercontent.com/58553711/201229737-be75e902-b5dc-4bed-bec3-ace613a7038c.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/201229811-95479e01-89bf-49b6-aee1-30efc2346595.png)